backend/donor-service/configs/constants.js

```diff
@@ -1,14 +1,24 @@
 const GENDER_MAP = {'M': 'Male', 'F': 'Female', 'T': 'Transgender'}
+const PLEDGE_STATUS = { 
+    "PLEDGED": "0", 
+    "UNPLEDGED": "1",
+    "NOTPLEDGED": "2"
```

Comment:
Where are we using NOT PLEDGED

Reply:
This status is not being stored in the Redis for now as the current flow doesn't support it, we are doing the redis transactions only after we initaite a pledage. For now Im using it to send it as a status back to the UI at mobile verificication step. And added it so that we can use it in future if needed.